app/assets/javascripts/arclight/collection_navigation.js

```diff
@@ -60,21 +60,23 @@
     }
   };
 
-  Blacklight.onLoad(function () {
+  Blacklight.onLoad(() => {
     'use strict';
 
-    $('[data-controller="arclight-contents"]').each(function (i, element) {
-      CollectionNavigation.init(element); // eslint-disable-line no-undef
-    });
+    document.querySelectorAll('[data-controller="arclight-contents"]').forEach((element) => {
+      CollectionNavigation.init(element)
+    })
 
-    $('[data-controller="arclight-contents"]').on('navigation.contains.elements', function (e) {
-      $(e.target).find('.collapse').on('show.bs.collapse', function (ee) {
-        var $newTarget = $(ee.target);
-        $newTarget.find('[data-controller="arclight-contents"]').each(function (i, element) {
-          CollectionNavigation.init(element); // eslint-disable-line no-undef
-        // Turn off additional ajax requests on show
-          $newTarget.off('show.bs.collapse');
```

Comment:
just for my own understanding — does `{ once: true }` achieve the same effect as using jQuery's `.off()` here?

Reply:
I believe it does. We have set up a listener that removes itself after the first time that it's called.